# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/TypedArrayObject.h
# Commit: 77ff4529ea3e
# Full Hash: 77ff4529ea3eb01ca0013bd4747979c226fb617f
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1720422
# File Overlap Count: 2
# Description:
#   Bug 1720422 - Expose a templatized TypedArray<T> class in JSAPI r=jonco,nbp
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119853
# ==============================================================================

diff -r 16fa627201d1 -r 77ff4529ea3e js/src/vm/TypedArrayObject.h
--- a/js/src/vm/TypedArrayObject.h	Thu Aug 12 20:28:53 2021 +0000
+++ b/js/src/vm/TypedArrayObject.h	Thu Aug 12 20:44:14 2021 +0000
@@ -21,19 +21,6 @@
 #include "vm/JSObject.h"
 #include "vm/SharedArrayObject.h"
 
-#define JS_FOR_EACH_TYPED_ARRAY(MACRO) \
-  MACRO(int8_t, Int8)                  \
-  MACRO(uint8_t, Uint8)                \
-  MACRO(int16_t, Int16)                \
-  MACRO(uint16_t, Uint16)              \
-  MACRO(int32_t, Int32)                \
-  MACRO(uint32_t, Uint32)              \
-  MACRO(float, Float32)                \
-  MACRO(double, Float64)               \
-  MACRO(uint8_clamped, Uint8Clamped)   \
-  MACRO(int64_t, BigInt64)             \
-  MACRO(uint64_t, BigUint64)
-
 namespace js {
 
 /*
